src/wazuh_db/schema_upgrade_v11.sql

```diff
@@ -0,0 +1,33 @@
+/*
+ * SQL Schema for upgrading databases
+ * Copyright (C) 2015, Wazuh Inc.
+ *
+ * Febrary 1, 2023.
+ *
+ * This program is a free software, you can redistribute it
+ * and/or modify it under the terms of GPLv2.
+*/
+
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS sys_hwinfo_tmp (
+    scan_id INTEGER,
+    scan_time TEXT,
+    board_serial TEXT,
+    cpu_name TEXT,
+    cpu_cores INTEGER,
+    cpu_mhz REAL,
+    ram_total INTEGER,
+    ram_free INTEGER,
+    ram_usage INTEGER,
+    checksum TEXT NOT NULL CHECK (checksum <> ''),
+    PRIMARY KEY (scan_id, board_serial)
+);
+
+INSERT INTO sys_hwinfo_tmp SELECT * FROM sys_hwinfo;
```

Comment:
Please, consider using the following expression when inserting the pre-existent data into the new table (see the rest of the schemas)

```
 INSERT INTO sys_hwinfo_tmp SELECT CASE WHEN checksum <> '' THEN checksum ELSE 'legacy' END AS checksum ...
```

Reply:
It seems unnecessary in this update because restrictions are being removed and therefore the existing data will always be valid and with respect to checksum it could not be null because there is a check that prevents it.
In order to the rest, perhaps I can expand why.

Reply:
Ok that's right, and thx for the explanation.